debug/debug.ts

```diff
@@ -32,8 +32,11 @@ Logger.activeLogLevel = LogLevel.Info;
 // add your debugging imports here and prior to submitting a PR git checkout debug/debug.ts
 // will allow you to keep all your debugging files locally
 // comment out the example
-import { Example } from "./example";
-Example();
+// import { Example } from "./example";
+// Example();
+
+import { TestCase } from "./debug-search";
```

Comment:
Can you revert the debug.ts file so your local changes aren't included?

`git checkout debug/debug.ts` should do it.